eli5/permutation_importance.py

```diff
@@ -0,0 +1,61 @@
+"""
+A module for computing feature importances by measuring how score decreases
+when a feature is not available.
+
+A similar method is described in Breiman, "Random Forests", Machine Learning,
+45(1), 5-32, 2001 (available online at
+https://www.stat.berkeley.edu/%7Ebreiman/randomforest2001.pdf), with an
+application to random forests. It is known in literature as
+"Mean Decrease Accuracy (MDA)" or "permutation importance".
+"""
+from __future__ import absolute_import
+
+import numpy as np  # type: ignore
+from sklearn.utils import check_random_state  # type: ignore
+
+
+def iter_shuffled(X, columns_to_shuffle=None, pre_shuffle=False,
```

Comment:
Is there a reason to offer this feature, but not to use it? Would getting the score as the mean of 3 pre-shuffled runs be reasonably efficient and more robust? Is it hard to measure or estimate how much lower the quality of your importance estimates are with `pre_shuffle=True`?